browser/brave_drm_tab_helper.cc

```diff
@@ -42,3 +42,5 @@ void BraveDrmTabHelper::OnWidevineKeySystemAccessRequest() {
   Browser* browser = chrome::FindBrowserWithWebContents(web_contents());
   browser->window()->UpdateToolbar(web_contents());
 }
+
+WEB_CONTENTS_USER_DATA_KEY_IMPL(BraveDrmTabHelper)
```

Comment:
weird, they removed this stuff and now added it back?

Reply:
Yep unresolved externs without them.